Linux/audit.sh

```diff
@@ -128,7 +157,11 @@ m)
 
 s)
     printf "Backing up MYSQL databases and config files\n"
-    
+    # Config File Backups
+    paths=("~/var/www/html" "~/etc/nginx" "pam")  # add other files that need backup here
```

Comment:
nitpick, put all const vars at top if possible, and make the name all caps and a name that isn't confusing such as COMMON_CONFIG_PATHS

Reply:
dude this script is filled with so much shit code i dont see the point in making it look nice now lmao